Additions to the README (#1)
cobaltcore-dev/.github: profile/README.md

Please see the linked issues for more details.

## :bear: Features

- **Automated Lifecycle:** The lifecycle for compute, AI, storage, and network hardware is fully automated using the IronCore Metal API and Kubernetes-based operators. 
- **Gardener extensions:** Gardener and GardenLinux extensions provide coordinated control-plane maintenance for reliable updates of the hypervisor and storage fleet. 
- **Greenhouse Integration:** Operational concerns are seamlessly integrated with the [Greenhouse Operations Platform](https://cloudoperators.github.io/greenhouse) and Heureka Security Posture Management.

## :busts_in_silhouette: Get Involved

Thank you for considering to contribute to our project.
To become an excellent contributor, check out our [contribution guidelines](https://github.com/cobaltcore-dev/rook/blob/master/CONTRIBUTING.md) and our [open issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Acobaltcore-dev+archived%3Afalse+).

## :blue_heart: Code of Conduct

To facilitate a nice environment for all, check out [our Code of Conduct](https://github.com/cobaltcore-dev/.github/blob/main/CODE_OF_CONDUCT.md).

## :books: Learn More

> To learn more about IronCore, check out the official documentation which is added soon!

Reviewer: ```suggestion
To learn more about CobaltCore, check out the official documentation which will be added soon!
```

Reviewer: IronCore?
